docs/schemas/itemMeta.json

```diff
@@ -0,0 +1,996 @@
+          "enum": ["date"]
+        },
+        "oldestDateAllowed": {
+          "description": "Oldest date that can be selected",
+          "type": "string"
+        },
+        "newestDateAllowed": {
+          "description": "Newest date that can be selected",
+          "type": "string"
+        }
+      }
+    },
+
+    "tas-time": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-time",
+      "title": "tas-time",
+      "description": "A time item HH:MM:SS-HH:MM",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["time"]
+        },
+        "oldestTimeAllowed": {
+          "description": "Oldest time that can be selected",
+          "type": "string"
+        },
+        "newestTimeAllowed": {
+          "description": "Newest time that can be selected",
+          "type": "string"
+        }
+      }
+    },
+
+    "tas-datetime": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-datetime",
+      "title": "tas-datetime",
+      "description": "A date item YYYY-MM-DDTHH:MM:SS-HH:MM",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["datetime"]
+        },
+        "oldestDateAllowed": {
+          "description": "Oldest datetime that can be selected",
+          "type": "string"
+        },
+        "newestDateAllowed": {
+          "description": "Newest datetime that can be selected",
+          "type": "string"
+        }
+      }
+    },
+
+    "tas-salary": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-salary",
+      "title": "tas-salary",
+      "description": "A salary question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["salary"]
+        },
+        "payBands": {
+          "description": "List of options if freeform input not allowed",
+          "type": "array",
+          "items": { "type": "string" }
+        },
+        "maximumDigitals": {
+          "description": "Max number characters",
+          "type": "number",
+          "default": 15
+        }
+      }
+    },
+
+    "tas-info": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-info",
+      "title": "tas-info",
+      "description": "An info item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["info"]
+        },
+        "name": {
+          "description": "Name to be used to ID heading",
+          "type": "string"
+        },
+        "text": {
+          "description": "Text for info",
+          "type": "string"
+        }
+      }
+    },
+
+    "tas-heading": {
```

Comment:
This is a more grunty question we'll need to discuss - should this be a question (ugly) or should it be a higher level thing that contains a bunch of questions?